includes/templates/template_default/templates/tpl_advanced_search_default.php

```diff
@@ -37,11 +37,13 @@
 <br class="clearBoth" />
 </fieldset>
 
+<?php if (!isset($skip_manufacturers) || $skip_manufacturers != true) { ?>
 <fieldset class="floatingBox forward">
     <legend><?php echo ENTRY_MANUFACTURERS; ?></legend>
     <?php echo zen_draw_pull_down_menu('manufacturers_id', zen_get_manufacturers(array(array('id' => '', 'text' => TEXT_ALL_MANUFACTURERS)), PRODUCTS_MANUFACTURERS_STATUS), $sData['manufacturers_id'], 'id="searchMfgId" aria-label="' . PLEASE_SELECT . '"'); ?>
```

Comment:
If this code-block is skipped, does the absence of the entire `manufacturers_id` field create any issues in the processing of the search form-fields?

Reply:
Nope no issues.